docs/variants/asus_kgpe_d16/dasharo-compatibility/flashrom_wp_test_specification.md

```diff
@@ -0,0 +1,203 @@
+# Flashrom Write Protection – Test Specification
+
+## Test cases
+
+### BIOS WP enable test
+
+#### Test description
+
+The test checks if it's possible to set the WP and whether it actually works
+
+#### Test configuration data
+
+1. `FIRMWARE` = coreboot
+
+2. `OPERATING_SYSTEM` = Ubuntu 20.04
+
+#### Test setup
+
+1. Proceed with the [Generic test setup: firmware](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#firmware)
+
+2. Proceed with the [Generic test setup: OS installer](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#os-installer)
+
+3. Proceed with the [Generic test setup: OS installation](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#os-installation)
+
+4. Proceed with the [Generic test setup: OS boot from disk](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#os-boot-from-disk)
+
+5. Check if flashrom and cbfstool are installed:  
```

Comment:
I think, that we don't need to describe all of the process (checking and installing). We might only write, that the test taker should install flashrom and cbfstool and how to do it.